packages/server/src/mL/Docker.ts

```diff
@@ -118,17 +118,17 @@ export default class Docker {
    * @param image The image to base this container on
    * @param ports The ports to expose
    */
-  async createContainer(project: Project, image: DockerImage, ports: [string?] = []): Promise<Container> {
+  async createContainer(project: Project, id: string, image: DockerImage, ports: [string?] = []): Promise<Container> {
     console.info(`${project.id}: Launching container ${image.name}`);
 
     const options: ContainerCreateOptions = {
       Cmd: [Docker.containerProjectPath(project)],
       Image: `${image.name}:${image.tag}`,
-      name: `wpilib-${image.name.replace(/\//g, "_")}-${project.id}`,
+      name: `wpilib-${image.name.replace(/\//g, "_")}-${id}`,
```

Comment:
Although very unlikely, you can have a collision here that is not protected by the database. The name needs to include unique information to prevent collisions. Happy to talk via Slack about it but I think we either need a note here saying that we are taking this risk or write something to ensure unique docker container names.